src/components/categoriess-container/categoriess-container.component.scss

```diff
@@ -0,0 +1,92 @@
+.categories-container {
+  padding-left: 20vw;
+  padding-right: 20vw;
+  display: grid;
+  grid-template-columns: repeat(6, 1fr);
+  grid-gap: 3vw;
+  place-items: center;
+}
+
+.custom-button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 2vw;
+  width: 5vw;
+  height: 5vw;
+  background-color: blanchedalmond;
+  border: none;
+  border-radius: 1vw;
+  cursor: pointer;
+}
+
+.centered-image {
+  max-width: 100%;
+  max-height: 100%;
+  object-fit: cover;
+}
+
+@media only screen and (max-width: 740px) {
+  .categories-container {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-gap: 1vw;
+    place-items: center;
+
+  }
+
+  .custom-button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 16vw;
+    height: 16vw;
+    background-color: blanchedalmond;
+    border: none;
+    border-radius: 0.5vw;
+    cursor: pointer;
+    padding: 5vw;
+
+  }
+
+  .centered-image {
+    max-width: 100%;
+    max-height: 100%;
+    object-fit: cover;
+  }
+
+
+}
+
+
+@media only screen and (max-width: 360px) {
```

Comment:
do you delete the scss styles that you don't use, or because you need 3 times the same classes?

Reply:
Yes, I need them for the 3 different resolutions those we support

Reply:
but in the 3 .centered-image is the same, 2 are unnecessary, delete them.

Reply:
yes, with the new commit for some resizes they are now different, thank you